networkx/utils/backends.py

```diff
@@ -1437,12 +1458,19 @@ def __call__(self, /, *args, backend=None, **kwargs):
                 "`nx.config.backend_priority`, or you "
                 "may specify a backend to use with the `backend=` keyword argument."
             )
+        if self._is_stub:
+            extra = (
+                " This function is a stub, which means it is included in NetworkX as "
```

Comment:
Similar wording elsewhere to just remove the text about "stub"
```suggestion
                " This function is included in NetworkX as "
```